### PR TITLE
Solution5

### PR DESCRIPTION
**Pull request by Viktoriia17051991**

### CODE REVIEW

**Review by irini132 — COMMENTED** [review]
Please add more tests: 
- should set firstName if it is omitted
- should work with 2 elements

**Review by Ivan-Shvorak — CHANGES_REQUESTED** [review]
Please send pull request to review

File: src/restoreNames.test.js [hunk 1]
```diff
@@ -1,11 +1,36 @@
 'use strict';
 
 describe('restoreNames', () => {
```
Comment (4ksandrey) [inline src/restoreNames.test.js:3]: Add more tests, for example, what about if user empty, if first name is exists (shouldn't set first name) 
Also remove comments)

**Review by 4ksandrey — CHANGES_REQUESTED** [review]

**Review by 4ksandrey — APPROVED** [review]
Well done
